host_guest/SAMPLing/absolute_submission_example.txt

```diff
@@ -0,0 +1,88 @@
+OA-G6-2, -2.90,, -4.15,, -3.21,, -2.64,, -4.10,, -3.42,, -1.99,, -2.25,, -3.62,, -3.79,, -3.07,, -3.68,, -3.26,, -1.23,, -3.44,, -2.94,, -2.73,, -2.51,, -3.42,, -3.95,, -3.59,, -1.44,, -3.82,, -2.02,, -2.69,, -4.08,, -2.48,, -3.02,, -2.59,, -2.89,, -3.58,, -2.65,, -3.15,, -3.03,, -4.04,, -2.90,, -2.99,, -3.58,, -2.23,, -2.40,, -2.72,, -2.86,, -4.02,, -3.39,, -3.73,, -2.19,, -2.98,, -2.65,, -3.73,, -3.18,, -1.85,, -2.47,, -4.30,, -2.84,, -2.09,, -3.00,, -3.15,, -2.78,, -2.73,, -3.56,, -3.40,, -3.44,, -3.17,, -2.52,, -3.19,, -3.58,, -3.31,, -4.45,, -3.35,, -2.31,, -2.58,, -1.87,, -2.09,, -1.97,, -3.89,, -3.57,, -2.38,, -3.85,, -3.91,, -2.70,, -3.13,, -2.84,, -1.79,, -2.66,, -3.20,, -2.70,, -3.64,, -3.04,, -3.14,, -4.01,, -3.85,, -2.50,, -2.70,, -3.25,, -3.53,, -2.41,, -2.38,, -2.98,, -2.36,, -3.41,
+OA-G6-3, -1.74,, -3.79,, -2.04,, -3.33,, -2.69,, -3.02,, -3.47,, -3.45,, -3.02,, -3.46,, -2.94,, -3.68,, -2.07,, -2.97,, -3.51,, -2.85,, -3.64,, -3.21,, -3.80,, -2.77,, -3.40,, -3.13,, -2.90,, -3.63,, -3.30,, -3.19,, -3.64,, -2.12,, -4.65,, -3.52,, -3.76,, -3.41,, -3.67,, -2.89,, -3.69,, -2.64,, -2.17,, -2.89,, -2.65,, -2.37,, -4.33,, -3.11,, -3.12,, -2.75,, -2.47,, -3.97,, -2.59,, -2.40,, -3.32,, -3.06,, -3.70,, -2.08,, -3.17,, -3.62,, -2.30,, -2.61,, -2.20,, -3.66,, -3.70,, -2.71,, -2.39,, -3.68,, -3.06,, -3.33,, -4.26,, -2.41,, -2.26,, -2.84,, -3.37,, -4.05,, -2.09,, -2.13,, -3.62,, -3.02,, -4.34,, -4.28,, -3.37,, -3.54,, -3.26,, -1.52,, -2.37,, -3.49,, -3.71,, -3.72,, -2.08,, -2.68,, -3.37,, -3.63,, -2.93,, -3.01,, -2.53,, -3.05,, -4.13,, -2.36,, -2.45,, -3.96,, -2.47,, -2.95,, -2.29,, -4.09,
+OA-G6-4, -3.45,, -2.74,, -3.76,, -3.52,, -3.24,, -2.23,, -3.77,, -3.35,, -2.45,, -4.36,, -2.09,, -4.23,, -3.33,, -3.84,, -2.93,, -2.99,, -1.89,, -3.24,, -2.64,, -2.94,, -3.78,, -3.06,, -2.52,, -3.17,, -2.35,, -3.66,, -2.41,, -3.34,, -2.86,, -2.07,, -4.38,, -2.28,, -4.18,, -3.77,, -2.97,, -2.36,, -3.50,, -4.24,, -2.60,, -2.49,, -3.10,, -1.98,, -3.93,, -3.49,, -2.72,, -3.47,, -2.86,, -3.42,, -2.73,, -2.67,, -2.87,, -2.57,, -2.63,, -4.20,, -2.23,, -3.33,, -2.44,, -2.92,, -3.56,, -3.18,, -2.04,, -3.05,, -2.93,, -2.62,, -3.99,, -1.28,, -1.45,, -2.38,, -1.93,, -2.82,, -2.56,, -3.02,, -3.03,, -3.40,, -3.03,, -3.27,, -2.17,, -2.98,, -2.42,, -1.47,, -2.14,, -3.37,, -2.85,, -3.39,, -3.37,, -3.49,, -3.52,, -2.49,, -2.35,, -3.88,, -3.18,, -3.24,, -3.73,, -2.57,, -2.86,, -3.43,, -3.19,, -3.72,, -3.65,, -2.84,
+#
+# Each line should include the following four elements in this order: system ID, total number of energy evaluations,
```

Comment:
"Each line of the COST section, below..."? Otherwise the context is a little confusing.